pages/common/mmdc.md

```diff
@@ -0,0 +1,17 @@
+# mmdc
+
+> CLI for mermaid, a diagram generation tool with a domain-specific language.
+> A mermaid definition file is taken as input and a svg, png, or pdf file is generated as output.
+> More information: <https://mermaid-js.github.io/mermaid/>.
+
+- Convert file to the specified format (automatically determined from the file extension. svg, png, and pdf are supported):
```

Comment:
The supported file extensions are already being described in the description so I don't think is necessary to write it again. 
```suggestion
- Convert a file to the specified format (automatically determined from the file extension):
```